src/app/presentation/introjs-directive/introjs.directive.ts

```diff
@@ -8,12 +8,16 @@ import {introJs} from 'intro.js/intro.js';
 export class IntrojsDirective implements OnInit {
 
   constructor(public slide: SlideComponent) {
+    if (!localStorage.numTours) {
+      localStorage.numTours = 0;
+    }
   }
 
   ngOnInit() {
     this.slide.onActive.subscribe((active) => {
-      if (active) {
+      if (active && localStorage.numTours <= 2) {
```

Comment:
I think so but it seems unlikely; wasn't sure if a better way was to have each slide to add something to local storage; didn't know what would be a good way to do that through the shared directive.